SeleniumPracticalExercise/PageObjects/VerifyEmployee.cs

```diff
@@ -0,0 +1,69 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools.V125.CSS;
+using OpenQA.Selenium.DevTools.V125.Page;
+using SeleniumExtras.WaitHelpers;
+using SeleniumPracticalExercise.PageObjects.Common;
+using static SeleniumPracticalExercise.PageObjects.Common.BasePageLocal;
+
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class VerifyEmployee : BasePageLocal
+    {
+        private readonly By EmployeeId = By.CssSelector("div.oxd-grid-2 input");     // EMployee ID # Locator
```

Comment:
You don't need to comment each locator, etc. The variable name should be good enough to indicate what the variable stores. Please remove these.

Reply:
removed